src/main/io/osd.c

```diff
@@ -71,6 +71,10 @@
 #include "fc/rc_controls.h"
 #include "fc/runtime_config.h"
 
+#ifdef USE_OSD_ADJUSTMENTS
```

Comment:
Build flags should not be used for `#includes` so just use
```
#include "fc/rc_adjustments.h"
```

Reply:
done.